src/features/user/MFV2/ProfileLayout/ProfileLayout.module.scss

```diff
@@ -25,9 +25,8 @@
 
   .mapContainer {
     margin: 0;
-    padding: 0;
+    padding: 4px;
```

Comment:
In the mobile view, no padding/border should be seen around the map. The only white space that should be seen should be below the map stats.